saleor/account/error_codes.py

```diff
@@ -4,7 +4,7 @@
 class AccountErrorCode(Enum):
     ACTIVATE_OWN_ACCOUNT = "activate_own_account"
     ACTIVATE_SUPERUSER_ACCOUNT = "activate_superuser_account"
-    ASSIGN_NON_STAFF_MEMBER = "assign_non_staff_member"
+    CANNOT_ADD_AND_REMOVE = "cannot_add_and_remove"
```

Comment:
The last thing that I would change here is this code- I'm not sure it clearly explains what happened. I would change it to something like `DUPLICATED_INPUT_ITEM`.